### PR TITLE
Ag lite with Create ReactApp

### PR DESCRIPTION
**Pull request by Ashersam**
Worked on creating our Autographa-Lite using create react app. Please the code.

### CODE REVIEW

File: src/components/About.js [hunk 1]
```diff
@@ -18,7 +18,7 @@ var AboutUsModel = function(props) {
                 <Tab eventKey={1} title={<FormattedMessage id="label-overview-tab"/>}>
                     <div className="row">
                         <div className="col-xs-6">
-                            <img src="../assets/images/autographa_lite_large.png" className="img-circle" alt="Cinque Terre" width="215" height="200" />
+                            <img src={require("../assets/images/autographa_lite_large.png")} className="img-circle" alt="Cinque Terre" width="215" height="200" />
```
Comment (joelthe1) [inline src/components/About.js]: Let's fix the alt text.

File: src/components/Loader.js [hunk 2]
```diff
@@ -1,5 +1,6 @@
 import React from 'react';
```
Comment (joelthe1) [inline src/components/Loader.js]: Remove this change.

File: src/components/Navbar.js [hunk 3]
```diff
@@ -1,5 +1,6 @@
 import React from 'react';
 import { observer } from "mobx-react"
+import swal from 'sweetalert';
```
Comment (joelthe1) [inline src/components/Navbar.js]: Please explain your changes here.

File: src/components/ReferencePanel.js [hunk 4]
```diff
@@ -25,7 +25,6 @@ class ReferencePanel extends React.Component {
         });
     }
     render (){
-        const layout = AutographaStore.layout;
```
Comment (joelthe1) [inline src/components/ReferencePanel.js]: Why is this removed?

File: src/components/Search.js [hunk 5]
```diff
@@ -1,13 +1,13 @@
 import React from 'react';
-const Modal = require('react-bootstrap/lib/Modal');
 import RaisedButton from 'material-ui/RaisedButton';
-const FormGroup = require('react-bootstrap/lib/FormGroup')
 import TextField from 'material-ui/TextField';
 import {RadioButton, RadioButtonGroup} from 'material-ui/RadioButton';
 import { observer } from "mobx-react"
 import AutographaStore from "./AutographaStore";
-const db = require(`${__dirname}/../util/data-provider`).targetDb();
 import { FormattedMessage } from 'react-intl';
```
Comment (joelthe1) [inline src/components/Search.js]: These changes don't seem to make any difference. Please undo.

File: src/index.css [hunk 6]
```diff
@@ -0,0 +1,13 @@
+body {
```
Comment (joelthe1) [inline src/index.css]: Please see if this is required.

File: src/index.js [hunk 7]
```diff
@@ -0,0 +1,17 @@
+import React from 'react';
```
Comment (joelthe1) [inline src/index.js]: Please see if this is required.

File: src/logo.svg [hunk 8]
```diff
@@ -0,0 +1,7 @@
+<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 841.9 595.3">
```
Comment (joelthe1) [inline src/logo.svg]: Please see if this is required.

File: src/translations/i18n.js [hunk 9]
```diff
@@ -39,7 +54,8 @@ i18n.prototype.isRtl = function(){
 
 i18n.prototype.currentLocale = function() {
 	return loadedLanguage.then(function(res){
-		return res['0']
+		console.log(res);
```
Comment (joelthe1) [inline src/translations/i18n.js]: Remove the console.logs

File: src/views/index.html [hunk 10]
```diff
@@ -8,7 +8,7 @@
     <meta name="description" content="">
     <meta name="author" content="">
     <link rel="icon" href="../../favicon.ico">
-    <title>Autographa Lite</title>
+    <title>Autographa Live</title>
```
Comment (joelthe1) [inline src/views/index.html]: This is Lite.

File: public/index.html [hunk 11]
```diff
@@ -0,0 +1,42 @@
+<!DOCTYPE html>
```
[the remark this answers is not in the document]
Comment (Ashersam) [reply public/index.html]: Yes create react app framed in that way.. it will look for the index.html file inside the public folder. This directory created by default we just made changes in the file accordingly

File: src/components/ReferencePanel.js (hunk 4 above)
Comment (Ashersam) [reply src/components/ReferencePanel.js]: Some variables are been removed since its declared and not used.

File: src/components/Search.js (hunk 5 above)
Comment (Ashersam) [reply src/components/Search.js]: All these are required and not removed, as a part of standard formatting. Declaration has been made after the import libraries

File: src/logo.svg (hunk 8 above)
Comment (Ashersam) [reply src/logo.svg]: removed, as this is a default react icon

File: src/components/Navbar.js (hunk 3 above)
Comment (Ashersam) [reply src/components/Navbar.js]: We are using swal in the nav bar for diff check popup and HTML export... We have import before we use it and I'm not sure how it's working in our old app.

File: src/components/Loader.js (hunk 2 above)
Comment (Ashersam) [reply src/components/Loader.js]: reverted changes

File: src/index.css (hunk 6 above)
Comment (Ashersam) [reply src/index.css]: removed as we are using electron app. this is useful for browser view

File: src/index.js (hunk 7 above)
Comment (Ashersam) [reply src/index.js]: yes this the create react app recognizes path through index.js

File: src/translations/i18n.js (hunk 9 above)
Comment (Ashersam) [reply src/translations/i18n.js]: yes removed

File: src/views/index.html (hunk 10 above)
Comment (Ashersam) [reply src/views/index.html]: I have removed this file and added the contents of this HTML to index.html inside public folder since we cannot alter the path...

File: .gitignore [hunk 12]
```diff
@@ -1,58 +1,27 @@
-node_modules
```
[the remark this answers is not in the document]
Comment (Ashersam) [reply .gitignore]: yes I have created a superset of both old and new files

File: package.json [hunk 13]
```diff
@@ -1,19 +1,46 @@
 {
-  "name": "Autographa",
-  "productName": "Autographa Lite",
-  "description": "A Bible translation editor for everyone.",
```
[the remark this answers is not in the document]
Comment (Ashersam) [reply package.json]: Added description

File: package.json [hunk 14]
```diff
@@ -1,19 +1,46 @@
...
-  "version": "1.2.5",
-  "main": "src/app.js",
+  "name": "autographa-lite",
+  "version": "1.2.6-beta.1",
+  "main": "public/electron.js",
   "author": "Friends of Agape <mathew.joel@protonmail.com>",
-  "license": "MIT",
+  "license": "GPL-3.0",
```
[the remark this answers is not in the document]
Comment (Ashersam) [reply package.json]: Changed

File: package.json [hunk 15]
```diff
@@ -31,92 +58,38 @@
     },
     "publish": null
   },
-  "keywords": [
```
[the remark this answers is not in the document]
Comment (Ashersam) [reply package.json]: Reverted

File: src/components/About.js (hunk 1 above)
Comment (Ashersam) [reply src/components/About.js]: I have changed to Autographa Logo
